src/muiComponents/CircularProgress/CircularProgress.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { forwardRef } from 'react';
+import { default as MaterialUICircularProgress, CircularProgressProps } from '@material-ui/core/CircularProgress';
+
+type CircularProgressRef = HTMLElementTagNameMap['div'];
```

Comment:
Why not just use `HTMLDivElement` directly?

Reply:
Yeah, good idea :-) Thanks!